openslides_backend/action/actions/meeting/update.py

```diff
@@ -203,6 +203,15 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
                 )
             if instance["jitsi_domain"].strip().endswith("/"):
                 raise ActionException("It is not allowed to end jitsi_domain with '/'.")
+
+        if (
+            instance.get("start_time")
+            and not instance.get("end_time")
+            or not instance.get("start_time")
+            and instance.get("end_time")
```

Comment:
You also need to check with the database here. Add a test with the following: `start_time` and `end_time` are set to some values. A `meeting.update` sets `start_time` to `None` and does not touch `end_time`. I suppose the test will go through, but shouldn't.

Reply:
Add the test, it failed. Updated the mixin to handle with database cases here.